synced_versions_formulae.json

```
@@ -31,7 +31,7 @@
   ["easyrpg-player", "liblcf"],
   ["etl", "synfig"],
   ["file-formula", "libmagic"],
-  ["edencommon", "fb303", "fbthrift", "fizz", "folly", "mvfst", "proxygen", "wangle", "watchman"],
+  ["edencommon", "fatal", "fb303", "fbthrift", "fizz", "folly", "mvfst", "proxygen", "wangle", "watchman"],
```

```suggestion
  ["edencommon", "fb303", "fbthrift", "fizz", "folly", "mvfst", "proxygen", "wangle", "watchman"],
```
Let's not -- it doesn't matter if `fatal` is out of sync and it just makes these PRs bigger.

---

👍